core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -587,15 +588,16 @@ class KafkaController(val config: KafkaConfig, zkClient: KafkaZkClient, time: Ti
               case e: Throwable =>
                 error(s"Error completing reassignment of partition $tp", e)
                 // remove the partition from the admin path to unblock the admin client
-                removePartitionFromReassignedPartitions(tp)
+                partitionsToBeRemovedFromReassignment.add(tp)
             }
           }
         } else {
             error(s"Ignoring request to reassign partition $tp that doesn't exist.")
-            removePartitionFromReassignedPartitions(tp)
+          partitionsToBeRemovedFromReassignment.add(tp)
```

Comment:
Seems like the `error` line is not indented correctly.

Reply:
Thanks for catching this! It is fixed now.